ecosystem/sep-0010.md

```diff
@@ -82,7 +92,7 @@ Request Parameters:
 Name      | Type          | Description
 ----------|---------------|------------
 `account` | `G...` string | The stellar account that the wallet wishes to authenticate with the server
-`home_domain` | string | (optional) The [fully qualified domain name](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of the service requiring authentication. SEP-10 signing servers used to generate tokens for multiple different web services can use this parameter to create valid tokens for each service. If not provided by the client, a default should be used.
+`home_domain` | string | (optional) The home domain the client would like to authenticate for. Servers that generate tokens for multiple home domains can use this parameter to identify which home domain the client hopes to authenticate for. If not provided by the client, the server should assume a default for backwards compatibility with older clients.
```

Comment:
The only issue I have with this phrasing is that clients don't authenticate _for_ a home domain, I know what you're trying to say but it is a bit confusing because you're authenticating for a service on the TOML, not the home domain.

Reply:
I think both can be true. The client authenticates for the home domain and that authentication can be used with services referenced on the home domain. Is that accurate?

Reply:
That latter statement is accurate, but we shouldn't try to shorthand that full explanation by saying you authenticate for a home domain. The shorthand used doesn't imply the full explanation easily.

Reply:
When I read that phrase I think, "wait, I'm authenticating for a JWT to use with a Stellar service, what does the TOML home domain have to do with authentication?"

Reply:
I think the answer to the question about what odes the home domain have to do with it is answered further above in the abstract section where we now clearly call out the discovery flow.

How does this subtle change in the language sound to you? b22afa9

Reply:
I like that distinction, since the SDK's are in fact verifying the home domain, which is authenticating in a way.